lib/ddtrace/contrib/mysql2/client.rb

```diff
@@ -0,0 +1,41 @@
+require 'ddtrace/ext/sql'
+require 'ddtrace/ext/app_types'
+
+module Datadog
+  module Contrib
+    module Mysql2
+      # Mysql2::Client patch module
+      module Client
+        module_function
+
+        def included(base)
+          base.send(:prepend, InstanceMethods)
```

Comment:
Thanks for refactoring your code to modules.  It looks really good!.
However unluckily we do also support Ruby 1.9.3 where "#prepend" is not supported.

[I just wrote this "guide"](https://github.com/DataDog/dd-trace-rb/blob/feature/document_prepend_1_9_3_compatibility/docs/PrependCompatibility.md) how prepend functionality can be replicated in Ruby 1.9.3. 

In short It requires some boilerplate adding two ifs and a Compatibility module to dispatch `super` to the right method. But it will allow us easier transition to using only prepend in the future.
/cc @delner

Reply:
I might suggest not supporting 1.9.3 at all, unless you need it. We are planning at dropping support for it in the future, and many of our new integrations already do not support 1.9.3. So if it's easier or preferable, you could stick with the `prepend` implementation and simply skip patching for 1.9.3. The `sequel` implementation should have an example of this compatibility check.

Reply:
Supporting < 2.0 seems “altruistic” at this point. Please decide amongst yourselves and I’ll act accordingly.

Reply:
I think I’ll add the 1.9 support. I want this merged. :)